packages/terra-form-validation/src/terra-dev-site/doc/form-validation/NumericValidation.6.doc.mdx

```diff
@@ -0,0 +1,10 @@
+import { Badge } from 'terra-form-validation/package.json?dev-site-package';
+
+import NumericValidation from '../example/NumericValidation?dev-site-example';
+
+<Badge />
+
+# Making a Numeric Input With Minimum, Maximum, and Precision Validations
+
+## Example 
+<NumericValidation title='Numeric Validations' />
```

Comment:
We can move this example to the main [Form Validation Page](https://terra-framework-deploy-pr-1046.herokuapp.com/components/terra-form-validation/form-validation/form-validation) to avoid a separate Page just for one example.

Reply:
Changed e52f755